devops/providers/azure-devops/templates/infrastructure/scripts/tf-apply-plan.yml

```diff
@@ -56,4 +57,4 @@ steps:
         TF_PLAN_FILE="${TF_WORKSPACE_NAME}_plan.out"
         TF_CLI_ARGS=${TF_CLI_ARGS:-}
 
-        terraform apply $TF_CLI_ARGS -auto-approve $TF_PLAN_FILE
+        terraform apply $TF_CLI_ARGS -input=false -auto-approve $TF_PLAN_FILE
```

Comment:
Why do we have `auto-approve` flag set here?

Reply:
This is because these steps run 'headless' via Az Pipeline Jobs. (there is not opportunity for a use to type "yes" into the `tf apply` STDIN via Az Pipelines). The "manual approval before `tf apply`" requirement is satisfied by storing the plan output between the `tf plan` and `tf apply` job steps, inserting a manual approval step (via Az Pipeline), and having he review look at that plan output json file.